collector/ibmmq/config.yaml

```diff
@@ -0,0 +1,23 @@
+receivers:
+  prometheus/ibmmq:
+    use_start_time_metric: false
+    start_time_metric_regex: '^(.+_)*process_start_time_seconds$'
+    config:
+      scrape_configs:
+        - job_name: 'ibmq-scraper'
+          scrape_interval: 5s
+          metrics_path: "/metrics"
+          static_configs:
+            - targets: [":9157"]
+exporters:
+  logging:
+    logLevel: debug
+  otlp:
+    endpoint: ingest.lightstep.com:443
+    headers:
+      - lightstep-access-token: "eyJhbGciOiJIUzI1NiIsImtpZCI6IjIwMTktMDMtMDEiLCJ0eXAiOiJKV1QifQ.eyJzY3AiOnsicm9sZSI6ImU1MTI5ODNiLTFjYjktMTFlOC05M2Y1LTQyMDEwYWYwMGFkNiJ9LCJ2ZXIiOjEsImRlYnVnIjp7Im9yZyI6IkxpZ2h0U3RlcCIsInJvbGUiOiJPcmdhbml6YXRpb24gQWRtaW4ifSwiYXVkIjoiYXBwLmxpZ2h0c3RlcC5jb20iLCJleHAiOjE2ODUwMzc0ODcsImp0aSI6InlhM25uemVoenlyaW1jZnpuM29ya3AyMnh2cDNqY2JuY3F3anpwaHprNTJ3bmdubCIsImlhdCI6MTY1MzUwMTQ4NywiaXNzIjoibGlnaHRzdGVwLmNvbSIsInN1YiI6IjEyOWY2ZWIzNGE3ZGJjZDcxZTI2NDM1Yzc4OSJ9.rjKjrFkO5nXJCKfdMiPnfUSgO8FeREULILpnEO1_Cko="
```

Comment:
I think we should leave this sort of token out of the repository. Is there a recommended solution for this in the other examples?

Reply:
We indeed do not show it, see https://github.com/lightstep/opentelemetry-examples/pull/102/files#diff-2806d525ff2fef4464988f7d5a3deca463163c46a0a1c017ae9e43bbffc02a62R14

Reply:
Ping @githubsands